src/neuvector/common/zarf.yaml

```diff
@@ -0,0 +1,35 @@
+kind: ZarfPackageConfig
+metadata:
+  name: uds-core-neuvector-common
+  description: "UDS Core Neuvector Common"
+  url: https://open-docs.neuvector.com/
+
+components:
+  - name: neuvector
+    description: "Deploy Neuvector"
+    required: true
+    charts:
+      - name: crd
+        url: https://neuvector.github.io/neuvector-helm/
+        version: 2.6.4
+        namespace: neuvector
+        gitPath: charts/crd
+      - name: uds-neuvector-config
+        namespace: neuvector
+        version: 0.1.0
+        localPath: ../chart
+      - name: core
+        url: https://neuvector.github.io/neuvector-helm/
+        version: 2.6.4
+        namespace: neuvector
+        gitPath: charts/core
+        valuesFiles:
+          - ../values/values.yaml
+      # Temporarily disabled until we can get the monitor chart working via https://github.com/neuvector/neuvector-helm/pull/355
+      # - name: monitor
+      #   url: https://neuvector.github.io/neuvector-helm/
+      #   version: 2.6.4
+      #   namespace: neuvector
+      #   gitPath: charts/monitor
+      #   valuesFiles:
+      #     - ../values/monitor-values.yaml
```

Comment:
I'm *okay* with this but in my head we could just put something like this in the registry1 values file:

```yaml
exporter:
  enabled: false
```

Ideally less churn/changes when we can re-enable it?

Reply:
thats fine. I did it this way because it seemed like the most obvious/apparent way to see that it was done and why. Removing the comments and updating the values with the comment on there as a note